include/fastdds/statistics/monitorservice_types.idl

```diff
@@ -91,7 +82,7 @@ module statistics {
 		INCONSISTENT_TOPIC,
 		LIVELINESS_LOST,
 		LIVELINESS_CHANGED,
-		DEADLINE_MISSED,
+		DEADLINE_sISSED,
```

Comment:
I think this was not intended. :sweat_smile:

Reply:
Good catch !